requirements.txt

```diff
@@ -21,7 +21,9 @@ pytest-trio
 pytest-cov
 trio
 trustme
+unasync==0.4.*
```

Comment:
That's a bit funny, but listing `unasync` in the `build-system.requires` entry doesn't *actually* install it in a way that's available to the Python interpreter. I don't know when/where those dependencies are installed exactly — probably managed by pip…